src/docs/sphinx/advancedExamples/validationStudies/proppantSlotTest/Example.rst

```diff
@@ -0,0 +1,292 @@
+
+
+---------------------------
+Proppant Transport solver
+---------------------------
+
+GEOSX is a multi-physics platform. Different combinations of physics solvers available in the code can be applied in different regions of the domain and be functional at different stages of the simulation. The ``Solvers`` tag in the XML file is used to list and parameterize these solvers.
+
+To specify a coupling between two different solvers, we define and characterize each single-physics solver separately.
+Then, we customize a *coupling solver* between these single-physics
+solvers as an additional solver.
+This approach allows for generality and flexibility in constructing multi-physics solvers.
+The order of specifying these solvers is not restricted in GEOSX.
+Note that end-users should give each single-physics solver a meaningful and distinct name, as GEOSX will recognize these single-physics solvers based on their customized names and create user-expected coupling.
+
+As demonstrated in this example, to setup a coupled proppant transport solver, we need to define three different solvers in the XML file:
+
+- the proppant transport solver for the fracture region, a solver of type ``ProppantTransport`` called here ``ProppantTransport`` (more information here: :ref:`ProppantTransport`),
+
+.. literalinclude:: ../../../../../../src/coreComponents/physicsSolvers/multiphysics/benchmarks/Proppant-transport-validation-case.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PROPPANT_PROPPANTSOLVER -->
+  :end-before: <!-- SPHINX_PROPPANT_PROPPANTSOLVER_END -->
+
+
+- the single-phase flow solver, a solver of type ``SinglePhaseProppantFVM`` called here ``SinglePhaseFVM``,
+
+.. literalinclude:: ../../../../../../src/coreComponents/physicsSolvers/multiphysics/benchmarks/Proppant-transport-validation-case.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PROPPANT_FLOWSOLVER -->
+  :end-before: <!-- SPHINX_PROPPANT_FLOWSOLVER_END -->
+
+
+- the coupling solver (``FlowProppantTransport``) that will bind the two single-physics solvers above, which is named as ``FlowProppantTransport``
+
+.. literalinclude:: ../../../../../../src/coreComponents/physicsSolvers/multiphysics/benchmarks/Proppant-transport-validation-case.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PROPPANT_COUPLEDSOLVER -->
+  :end-before: <!-- SPHINX_PROPPANT_COUPLEDSOLVER_END -->
+
+
+In this example, let us focus on the coupling solver.
+This solver (``FlowProppantTransport``) describe the coupling process between proppant and flow transport within the ``Fracture`` region. In this way, the two single-physics solvers (``ProppantTransport`` and ``SinglePhaseFVM``) will be simultaneously called and executed for solving this proppant transport problem.
+
+
+------------------------------
+Constitutive laws
+------------------------------
+
+For this slot test, 30/50 mesh proppant was injected via the three inlets and flowing through the slot for 30 seconds. 
```

Comment:
```suggestion
For this slot test, 30/50 mesh proppant is injected via the three inlets and flowing through the slot for 30 seconds. 
```
and I suggest explaining what 30/50 mesh proppant is (I admit I do not know what it means)